.travis.yml

```diff
@@ -1,39 +1,29 @@
 language: julia
-
 os:
-  - osx
   - linux
-
+  - osx
+  - windows
 julia:
-  - 1.0
-  - 1.1
   - 1.2
   - nightly
-
-# # Uncomment the following lines to allow failures on nightly julia
```

Comment:
Why remove?

Reply:
Should packages usually test against nightly?

Reply:
Yes, though it would make sense to allow failures on nightlies by default. At any rate it's better to show how to do it.